### PR TITLE
Add NCSS 2k16 memes

### PR DESCRIPTION
**Pull request by paked**
Thanks @tgsergeant for the opportunity to have my name put amongst the giants of memes.

I've painstakingly searched the NCSS 2k16 videos, scouring for some truly hot spicy memes. I believe I have found the best.

note: I've also taken some from previous years, as requested by other people.

Feel free to request any to be remove if they do not live up to the meme standards!

### CODE REVIEW

**Comment by iandrewt** [comment]
+1

**Comment by ghost** [comment]
+1

**Comment by Venergon** [comment]
+1

**Comment by gnustomp** [comment]
guys you're all :+1:'ing wrong.

File: static/js/main.js [hunk 1]
```diff
@@ -51,6 +58,7 @@ $(document).ready(function() {
       var pos = 0;
 
       document.addEventListener('keypress', function(e) {
+          console.log(e);
```
Comment (tgsergeant) [inline static/js/main.js:61]: ~~Please remove th~~lol i don't care

**Comment by tgsergeant** [comment]
let me tell you a fun story:

github delivers emails to my work account, because AFAIK you can't specify different emails for different orgs or anything fancy like that. So I wasn't going to see this storm of :+1:s until tomorrow morning.

Thanks to a [certain hshoebridge](https://twitter.com/hshoebridge/status/687221555688902656), though, i saw it now and get get these memes on the grill right away.

**Comment by tgsergeant** [comment]
:+1: :shipit: yolo merge

**Comment by tgsergeant** [comment]
rip mobile data caps, some of these gifs are pretty big

anyway, that's all deployed and ready to spice up a browser near you

what a memeball

**Pull request merged**